src/runtime/container/ndarray.cc

```diff
@@ -566,6 +566,35 @@ struct NDArray::Internal {
   }
 };
 
+NDArray NDArray::Reshape(std::vector<int64_t> shape) const {
```

Comment:
This interface needs to support three types: List/Tuple/Any. 
STL types do not necessarily support.

Reply:
added List/Tuple/Any/FTList